fcos-policy-engine/Cargo.toml

```diff
@@ -7,6 +7,7 @@ publish = false
 
 [dependencies]
 actix = "^0.9.0"
+actix-cors = "^0.2"
```

Comment:
I think this dependency is not required for `fcos-graph-builder` and `fcos-policy-engine` but it's also totally fine to add in for safety.

Reply:
Good catch! Dropped both.